src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -17,7 +17,7 @@ export class DictionaryModal extends React.Component {
         name: '',
         owner: '',
         description: '',
-        supported_locales: 'en, es',
+        supported_locales: 'en',
```

Comment:
Is this for only `English` and `Spanish` OR it can affect other languages

Reply:
It affects only dictionaries created in English anjd Spanish. I have tested out other languages and they are fine

Reply:
ok

Reply:
Shouldn't the fix have been to select one of the supported locales?

Reply:
Let me change the wording of the ticket to reflect this

Reply:
@dkayiwa I have made the relevant changes.

Reply:
To me, supported locales looks like should take a list of them.

Reply:
@dkayiwa Thanks for pointing that out.
The dictionary modal saves the language in the supported locales field. In state, the supported locales default value is "en, es" so whenever a language is saved its value is "en, es". So I changed the supported locales to a single language to fix that.

Reply:
Does that mean supported_locales cannot contain more than one locale?

Reply:
@dkayiwa It is true, the supported_locales can contain more than one locale.
I have gone through the documentation again and realised that instead of saving the dictionary language in the supported_locales field, we could instead save it in the default_locale field.
This enables us to leave the supported_locale as a list of languages and also have a designated field for the dictionary language as the default_locale.
I have also made the relevant changes to the code to reflect the above mentioned.